R/Reactant/build_tarballs.jl

```diff
@@ -292,7 +292,7 @@ for platform in platforms
         push!(cuda_deps, BuildDependency(PackageSpec(name="CUDNN_jll")))
         push!(cuda_deps, BuildDependency(PackageSpec(name="TensorRT_jll")))
         push!(cuda_deps, BuildDependency(PackageSpec(name="CUDA_full_jll")))
-        augment_platform_block=CUDA.augment
+        # augment_platform_block=CUDA.augment
```

Comment:
hm this is necessary, otherwise it will load the wrong binaries later.

Reply:
so ldd on our generated .so doesnt link to cuda (presumably static)

Reply:
And https://github.com/JuliaPackaging/Yggdrasil/blob/720d1ec37613e02974dd8e548c52bb790247f95c/platforms/cuda.jl#L13 is just a string.